app/home/homeController.js

```diff
@@ -168,6 +203,6 @@
         loadPosts();
         getInstitutions();
 
-        intervalPromise = $interval(loadPosts, 5000);
+        intervalPromise = $interval(loadPosts, 15000);
```

Comment:
Comenta essa linha. Coloca um FIXME comment.
Exemplo:
/**
              FIXME: First version doesn't treat the case in which the user is already
              the institution follower.
               @author: Maiana Brito 01/06/2017
**/